src/arkode/arkode_impl.h

```diff
@@ -355,8 +374,6 @@ struct ARKodeMemRec
   /* Time step data */
   realtype hin;                /* initial step size                        */
   realtype h;                  /* current step size                        */
-  realtype hmin;               /* |h| >= hmin                              */
```

Comment:
`hmin` is used in arkode_relaxation.c, so this being removed is now causing a compilation error.